pytest/lib/branches.py

```diff
@@ -54,7 +54,7 @@ def download_binary(branch):
 
 def prepare_ab_test(other_branch):
     compile_current()
-    if other_branch in ['master', 'beta', 'stable']:
+    if os.environ.get('BUILDKITE') and other_branch in ['master', 'beta', 'stable']:
```

Comment:
So, does this mean that downloading a prebuild binary makes sense only in the BuildKite environment? /cc @ailisp

Reply:
Yeah, on mac they don't quite work :)

Reply:
It also makes sense for local linux, and soon mac (#2299 ) I'll change this after #2299 being build for both master and beta.